### PR TITLE
Fix: Intezer URL analysis indicators parsing

### PR DESCRIPTION
**Pull request by davidt99**
## Contributing to Cortex XSOAR Content
Make sure to register your contribution by filling the [contribution registration form](https://forms.gle/XDfxU4E61ZwEESSMA)

**The Pull Request will be reviewed only after the contribution registration form is filled.**

## Status
- [ ] In Progress
- [x] Ready
- [ ] In Hold - (Reason for hold)

## Description
Parse URL indicators correctly

## Minimum version of Cortex XSOAR
- [x] 6.0.0
- [ ] 6.1.0
- [ ] 6.2.0
- [ ] 6.5.0

## Does it break backward compatibility?
   - [ ] Yes
       - Further details:
   - [x] No

## Must have
- [x] Tests
- [x] Documentation

### CODE REVIEW

**Comment by ShacharKidor** [comment]
Hi @davidt99, 
Thank you for contributing!

I can see that you are still committing changes, is the PR ready for review? 

In addition, how can I contact you? do you have a slack user?

**Comment by davidt99** [comment]
@ShacharKidor It's ready for review, but I'm having problems with build, I think there is an issue with the base image.
You can contact me via email or you can add me to XSOAR slack community and we can talk directly.

```
============================= test session starts ==============================
platform linux -- Python 3.10.1, pytest-7.0.1, pluggy-1.0.0 -- /home/circleci/project/venv/bin/python3
cachedir: .pytest_cache
rootdir: /home/circleci/project
plugins: freezegun-0.4.2, mock-3.7.0, requests-mock-1.9.3
collecting ... collecting 23 items / 1 error / 22 selected                                    collected 24 items / 1 error / 23 selected                                     

==================================== ERRORS ====================================
_ ERROR collecting Tests/scripts/infrastructure_tests/test_collect_tests_and_content_packs.py _
Tests/scripts/infrastructure_tests/test_collect_tests_and_content_packs.py:17: in <module>
    from Tests.scripts.collect_tests_and_content_packs import (
Tests/scripts/collect_tests_and_content_packs.py:19: in <module>
    from Tests.Marketplace.marketplace_services import get_last_commit_from_index
Tests/Marketplace/marketplace_services.py:26: in <module>
    import Tests.Marketplace.marketplace_statistics as mp_statistics
Tests/Marketplace/marketplace_statistics.py:6: in <module>
    from google.cloud.bigquery.client import Client
venv/lib/python3.10/site-packages/google/cloud/bigquery/__init__.py:35: in <module>
    from google.cloud.bigquery.client import Client
venv/lib/python3.10/site-packages/google/cloud/bigquery/client.py:60: in <module>
    from google.cloud.bigquery import _pandas_helpers
venv/lib/python3.10/site-packages/google/cloud/bigquery/_pandas_helpers.py:41: in <module>
    from google.cloud.bigquery import schema
venv/lib/python3.10/site-packages/google/cloud/bigquery/schema.py:19: in <module>
    from google.cloud.bigquery_v2 import types
venv/lib/python3.10/site-packages/google/cloud/bigquery_v2/__init__.py:23: in <module>
    from google.cloud.bigquery_v2 import types
venv/lib/python3.10/site-packages/google/cloud/bigquery_v2/types.py:23: in <module>
    from google.cloud.bigquery_v2.proto import encryption_config_pb2
venv/lib/python3.10/site-packages/google/cloud/bigquery_v2/proto/encryption_config_pb2.py:43: in <module>
    _descriptor.FieldDescriptor(
venv/lib/python3.10/site-packages/google/protobuf/descriptor.py:560: in __new__
    _message.Message._CheckCalledFromGeneratedFile()
E   TypeError: Descriptors cannot not be created directly.
E   If this call came from a _pb2.py file, your generated code is out of date and must be regenerated with protoc >= 3.19.0.
E   If you cannot immediately regenerate your protos, some other possible workarounds are:
E    1. Downgrade the protobuf package to 3.20.x or lower.
E    2. Set PROTOCOL_BUFFERS_PYTHON_IMPLEMENTATION=python (but this will use pure-Python parsing and will be much slower).
E   
E   More information: https://developers.google.com/protocol-buffers/docs/news/2022-05-06#python-updates
```

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
Hi @davidt99, 
Nice catch!

I added two little suggestions - please take a look. 

Regarding the build error, it might be related to the docker image, please try to run the following SDK command on the integration's YML file: 
`demisto-sdk format -ud <integration yml path>`

Don't hesitate to contact me with any questions :)

**Comment by davidt99** [comment]
@ShacharKidor I've got the same error, it's related to a protobuf that is unrelated to what I did. Can you assist?

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2.py [hunk 1]
```diff
@@ -506,10 +507,8 @@ def enrich_dbot_and_display_url_analysis_results(intezer_result, intezer_api):
         intezer_result['redirect_chain'] = redirect_chain
 
     if 'indicators' in intezer_result:
-        indicators: Dict[str, List[str]] = {}
+        indicators: Dict[str, List[str]] = defaultdict(list)
```
Comment (ShacharKidor) [inline Packs/Intezer/Integrations/IntezerV2/IntezerV2.py:510]: Why did you remove the fix you previously added here?

File: Packs/Intezer/ReleaseNotes/1_3_2.md [hunk 2]
```diff
@@ -0,0 +1,3 @@
+#### Integrations
+##### Intezer v2
+- Fix URL analysis indicators parsing.
```
Comment (ShacharKidor) [inline Packs/Intezer/ReleaseNotes/1_3_2.md:3]:
```suggestion
- Fixed an issue in URL indicators parsing.
```

**Review by ShacharKidor — CHANGES_REQUESTED** [review]
Hi @davidt99, 
I'll investigate the build error and will let you know as soon as possible. 

Your last changes in the python file confused me a bit - I think you accidentally removed the fix that was added to the indicators dictionary - can you please check that? 

Thanks.

File: Packs/Intezer/Integrations/IntezerV2/IntezerV2.py (hunk 1 above)
Comment (davidt99) [reply Packs/Intezer/Integrations/IntezerV2/IntezerV2.py:510]: After you comment, I just used a different solution to the issue which is default dict

Comment (ShacharKidor) [reply Packs/Intezer/Integrations/IntezerV2/IntezerV2.py:510]: I think I prefer the previous solution because using the 'defaultdict' package entails a change to the py3-tools docker image and I prefer not to do so. This may delay the PR unnecessarily.
Your previous solution is good enough.

Comment (davidt99) [reply Packs/Intezer/Integrations/IntezerV2/IntezerV2.py:510]: `defaultdict` is part of `collections` package which is a built-in package in python so new packages in the docker image 🙂

Comment (ShacharKidor) [reply Packs/Intezer/Integrations/IntezerV2/IntezerV2.py:510]: okay - good to know! :) 
I'll update you about the build issue on Sunday. Have a great weekend!

**Comment by ShacharKidor** [comment]
The same changes were moved to a different PR: [19239](https://github.com/demisto/content/pull/19239).
Closing the PR.
